editors/intellij/src/test/testData/basic-project/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "basic-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "license": "MIT",
+	"dependencies": {
+		"@biomejs/biome": "1.2.2"
```

Let's use pnpm workspaces. 
```suggestion
		"@biomejs/biome": "workspace:*"
```

And let's add the `src/test` folder in `pnpm-workspace.yml`.

Like this, we always use the local version. Doing so, would require building the npm package first and, hence, the binaries.

---

My very initial idea is to test on multiple package managers and maybe with monorepos
For now, I'll put pnpm on and plan the other cases better